eng/scripts/Language-Settings.ps1

```diff
@@ -88,20 +87,22 @@ function Publish-javascript-GithubIODocs ($DocLocation, $PublicArtifactLocation)
   $PublishedDocs = Get-ChildItem "$($DocLocation)/documentation" | Where-Object -FilterScript { $_.Name.EndsWith(".zip") }
 
   foreach ($Item in $PublishedDocs) 
-  {
-    $PkgName = "azure-$($Item.BaseName)"
-    # Override package name from other scope. We don't have package name to parse this out here.
-    if ($ArtifactToPackgeMap.ContainsKey($($Item.BaseName)))
-    {
-      $PkgName = $ArtifactToPackgeMap[$($Item.BaseName)]
-    }
-    Write-Host $PkgName
+  {    
     Expand-Archive -Force -Path "$($DocLocation)/documentation/$($Item.Name)" -DestinationPath "$($DocLocation)/documentation/$($Item.BaseName)"
     $dirList = Get-ChildItem "$($DocLocation)/documentation/$($Item.BaseName)/$($Item.BaseName)" -Attributes Directory
 
     if ($dirList.Length -eq 1)
     {
       $DocVersion = $dirList[0].Name
+      $parsedPackage = RetrieveReleasePackageInfo "NPM" $PublicArtifactLocation
```

Comment:
In this context I think you could directly call Get-javascript-PackageInfoFromPackageFile which lives in this file. You likely need the tag change in https://github.com/Azure/azure-sdk-for-js/pull/12501 as well.